currencies.php

```diff
@@ -0,0 +1,574 @@
+				'format' => 'RSD %s',
+			),
+			'SCR' => array(
+				'label'  => __( 'Seychelles Rupee', 'camptix' ),
+				'format' => 'SCR %s',
+			),
+			'SLL' => array(
+				'label'  => __( 'Leone', 'camptix' ),
+				'format' => 'SLL %s',
+			),
+			'SBD' => array(
+				'label'  => __( 'Solomon Islands Dollar', 'camptix' ),
+				'format' => 'SBD %s',
+			),
+			'SOS' => array(
+				'label'  => __( 'Somali Shilling', 'camptix' ),
+				'format' => 'SOS %s',
+			),
+			'LKR' => array(
+				'label'  => __( 'Sri Lanka Rupee', 'camptix' ),
+				'format' => 'LKR %s',
+			),
+			'SRD' => array(
+				'label'  => __( 'Surinam Dollar', 'camptix' ),
+				'format' => 'SRD %s',
+			),
+			'SZL' => array(
+				'label'  => __( 'Lilangeni', 'camptix' ),
+				'format' => 'SZL %s',
+			),
+			'TJS' => array(
+				'label'  => __( 'Somoni', 'camptix' ),
+				'format' => 'TJS %s',
+			),
+			'TZS' => array(
+				'label'  => __( 'Tanzanian Shilling', 'camptix' ),
+				'format' => 'TZS %s',
+			),
+			'TOP' => array(
+				'label'  => __( 'Pa’anga', 'camptix' ),
+				'format' => 'TOP %s',
+			),
+			'TTD' => array(
+				'label'  => __( 'Trinidad and Tobago Dollar', 'camptix' ),
+				'format' => 'TTD %s',
+			),
+			'UGX' => array(
+				'label'  => __( 'Uganda Shilling', 'camptix' ),
+				'format' => 'UGX %s',
+			),
+			'UAH' => array(
+				'label'  => __( 'Hryvnia', 'camptix' ),
+				'format' => 'UAH %s',
+			),
+			'UYU' => array(
+				'label'  => __( 'Peso Uruguayo', 'camptix' ),
+				'format' => 'UYU %s',
+			),
+			'UZS' => array(
+				'label'  => __( 'Uzbekistan Sum', 'camptix' ),
+				'format' => 'UZS %s',
+			),
+			'VUV' => array(
+				'label'  => __( 'Vatu', 'camptix' ),
+				'format' => 'VUV %s',
+			),
+			'VND' => array(
+				'label'  => __( 'Dong', 'camptix' ),
+				'format' => 'VND %s',
+			),
+			'YER' => array(
+				'label'  => __( 'Yemeni Rial', 'camptix' ),
+				'format' => 'YER %s',
+			),
+			'ZMW' => array(
+				'label'  => __( 'Zambian Kwacha', 'camptix' ),
+				'format' => 'ZMW %s',
+			),
+			'MRO' => array(
+				'label'  => __( 'Mauritanian Ouguiya', 'camptix' ),
+				'format' => 'MRO %s',
+			),
+		);
+	}
+
+	/**
+	 * Supported currencies are added via filter by different payment gateway addons/plugins
+	 * Addons should have `$supported_currencies` variable defined with list of currencies that they support in
+	 * ISO format
+	 *
+	 * @return array currency code of all supported currencies.
+	 */
+	private static function get_supported_currency_list() {
+		$supported_currencies = array();
+		return apply_filters( 'camptix_supported_currencies', $supported_currencies );
+	}
+
+	/**
+	 * @return array list of currencies with their labels, which are currently supported.
+	 */
+	public static function get_currencies() {
+		// from https://stackoverflow.com/a/4260168/1845153
+		$supported_currency_labels = array_intersect_key(
+			self::get_currency_list(), array_flip( self::get_supported_currency_list() ) );
```

Comment:
The spacing and lack of line breaks here makes it a bit hard to read at a glance. How about something like this:

```php
$supported_currency_labels = array_intersect_key(
	self::get_currency_list(),
	array_flip( self::get_supported_currency_list() )
);
```